runrole

```diff
@@ -61,7 +61,7 @@ fi
 #fi
 
 if [ "$REINSTALL" == "1" ]; then
-    if [ ! $1 == "internetarchive" ]; then # special handling
+    #if [ ! $1 == "internetarchive" ]; then # special handling
```

Comment:
This is a change @mitra42 wants? Removing the installed flag results in #1921 being reverted.

Reply:
Seems like a very positive change to standardize most all IIAB roles so implementers stay sane.

But @mitra42 is here with me in Arizona and we might call you later to see if other options are better...

Reply:
To be clear - I haven't requested any change - We finally got a way about two weeks ago to do an update. (It was there before, but without documentation weren't able to use it) 
```
cd /opt/iiab/iiab
git pull
./runrole --reinstall internetarchive
```

I'm fine if that changes to some other command, though something like ..

`ARGS="$ARGS -e reinstall=True" ` 

Plus whatever else is required (I'm sure this isn't all),  seems highly prone to mistyping, so I much prefer the former.